src/python/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "python",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
Looks like tests are failing with `configure: error: Invalid --with-openssl value`, can you help fix them?

Reply:
done. not failing now 👍